Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -2922,6 +2922,34 @@ bool AccessibilityNodeObject::canSetSelectedAttribute() const
     }
 }
 
+bool AccessibilityNodeObject::isListInstance() const
```

Comment:
Should this just be isAccessibilityList since it delegates to a function with that name?